mks/model/stuff.py

```diff
@@ -196,6 +205,12 @@ def get_persoon(self):
             if geslacht:
                 result['omschrijvingGeslachtsaanduiding'] = geslacht
 
+        # vertrokken onbekend waarheen
+        if result['emigratieLand'] == 0:
+            result['vertrokkenOnbekendWaarheen'] = True
```

Comment:
if not this then possibly: `result['vertrokkenOnbekendWaarheen'] = (result['emigratieLand'] == 0)`